run-tests-with-build-kogito-examples-parent/README.md

```diff
@@ -26,6 +26,13 @@ mvn clean verify \
 -Dtest.type=quickstarts-zip \
 -Ddownload.sources.url=http://link.to/quickstarts.zip
 ```
+### Scm-checkout testing
+```
+mvn clean verify \
+-Dtest.type=scm-checkout \
+-Drepository.url=https://github.com/kiegroup/kogito-examples \
+-Dsources.revision=1.11.x
+```
```

Comment:
Suggestion to add here the mention of platform parameters and why do we need it

-Dquarkus.platform.group-id=com.redhat.quarkus.platform \
-Dkogito.bom.group-id=com.redhat.quarkus.platform \
-...

Reply:
I've added a general section below - please see if it resolves the concern.